### PR TITLE
Rescue GibbonError

### PR DESCRIPTION
**Pull request by tonyvince**

### CODE REVIEW

File: app/jobs/mailchimp_inviter_job.rb [hunk 1]
```diff
@@ -2,15 +2,19 @@ class MailchimpInviterJob < ActiveJob::Base
...
-    )
+      gibbon.lists(list_id).members.create(
+        body: {
+          email_address: email,
+          status: 'subscribed',
+          merge_fields: { FNAME: first_name, LNAME: last_name }
+        }
+      )
+    rescue Gibbon::GibbonError=> e
+      logger.info "There is a problem with Mailchimp Invite: #{e.message}"
```
Comment (rickr) [inline app/jobs/mailchimp_inviter_job.rb]: Can this be changed `logger.error`

File: app/jobs/mailchimp_inviter_job.rb [hunk 2]
```diff
@@ -2,15 +2,19 @@ class MailchimpInviterJob < ActiveJob::Base
   queue_as :default
 
   def perform(email:, first_name:, last_name:)
-    gibbon = Gibbon::Request.new(api_key: OperationCode.fetch_secret_with(name: :mailchimp_api_key))
-    list_id = OperationCode.fetch_secret_with name: :mailchimp_list_id
+    begin
```
Comment (rickr) [inline app/jobs/mailchimp_inviter_job.rb]: This begin is redundant. You can remove it and place your rescue at the same level as the method definition.

File: app/jobs/mailchimp_inviter_job.rb [hunk 3]
```diff
@@ -2,15 +2,19 @@ class MailchimpInviterJob < ActiveJob::Base
...
-      }
-    )
+      gibbon.lists(list_id).members.create(
+        body: {
+          email_address: email,
+          status: 'subscribed',
+          merge_fields: { FNAME: first_name, LNAME: last_name }
+        }
+      )
+    rescue Gibbon::GibbonError=> e
```
Comment (rickr) [inline app/jobs/mailchimp_inviter_job.rb]: Can you add a space to the right of the hash rocket?

**Comment by tonyvince** [comment]
@rickr Changes suggested in your review has been done. Please take a look

**Comment by rickr** [comment]
Looks good. Thanks for the quick turn around!

**Pull request merged**